lib/spack/docs/conf.py

```diff
@@ -192,9 +187,9 @@ def setup(sphinx):
 # built documents.
 #
 # The short X.Y version.
-version = '.'.join(spack_version[:2])
+version = str(spack.spack_version.up_to(2))
 # The full version, including alpha/beta/rc tags.
-release = '.'.join(spack_version[:2])
+release = str(spack.spack_version.up_to(2))
```

Comment:
Should we use the full X.Y.Z version for this, or should we stick with X.Y?

Reply:
I think x.y.z is better